invoices/invoiceregistry.go

```diff
@@ -12,6 +14,11 @@ import (
 	"github.com/lightningnetwork/lnd/queue"
 )
 
+const (
+	// KeySendRecord is the custom record identifier for key send preimages.
+	KeySendRecord uint64 = 5482373484
```

Comment:
Perhaps this should instead live in the `record` package like all our "recognized" TLV types.

Reply:
Done. Reads nice: `record.KeySend`